src/types/v2_beatmap_set_download.ts

```diff
@@ -26,5 +26,5 @@ export interface types {
    * @param {boolean} no_video Download with or without video
    * @param {Function} callback function which is will be triggered on downloading progress
   */
-  (beatmapset: number, file_path: string, host_name: 'osu' | 'chimu' | 'beatconnect' | 'sayobot' | 'nerinyan' | '' , no_video: boolean, callback?: Function): Promise<string>;
```

Comment:
revert back to `string | null`. Because it'll brake someone else code, if they are using 2.x.x version. I'll change it on 3.0 tho